core/src/config.jl

```diff
@@ -134,12 +134,12 @@ end
     endtime::DateTime
 
     # optional, when Config is created from a TOML file, this is its directory
-    relative_dir::String = "."  # ignored(!)
+    relative_dir::String = "."
     input_dir::String = "."
-    results_dir::String = "."
+    results_dir::String = "results"
```

Comment:
Here you need to remove the default values, right?

Reply:
This is the same as the results fields. Let's see if I can fix them, as in not set by toml.